hack/e2e/run-rp-and-e2e.sh

```diff
@@ -65,9 +65,31 @@ register_sub() {
 }
 
 run_e2e() {
-    RESOURCEGROUP=$ARO_RESOURCEGROUP make test-e2e
+    export E2E_ENTRYPOINT="docker run --privileged \
+                --network host \
+                --env-file <(env | egrep 'AZURE|E2E|ARO') \
+                --env LOCATION \
+                --env CI \
+                --env RESOURCEGROUP=$ARO_RESOURCEGROUP \
+                --env CLUSTER=$CLUSTER \
+                --env RP_MODE \
+                --entrypoint e2e.test $ARO_IMAGE -test.timeout 180m -test.v -ginkgo.v"
+    RESOURCEGROUP=$ARO_RESOURCEGROUP E2E_ENTRYPOINT=$E2E_ENTRYPOINT make test-e2e
 }
 
+get_e2e_image_version(){
+    curl -s -w "\n%{http_code}" "https://${ARO_VERSION_SA}.blob.core.windows.net/rpversion/${LOCATION}" | {
```

Comment:
where does ARO_VERSION_SA come from?  It is not documented anywhere in our codebase.

Reply:
Added few reference to this